tensorboard/webapp/notification_center/_views/notification_center_component.scss

```diff
@@ -12,3 +12,54 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+
+::ng-deep .notification-menu.mat-menu-panel {
+  width: 350px;
+  font-size: 10px;
+}
+
+.menu-item {
+  padding: 17px 17px 0 17px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.menu-item:last-child {
+  border-bottom: 0;
+}
+
+.notification-menu {
+  font-size: 10px;
+}
+
+.category-icon {
+  height: 16px;
+  vertical-align: middle;
+  width: 16px;
+  color: #4285f4;
+  margin-right: 6px;
+}
+
+.content-wrapper {
+  margin-left: 23px;
+  margin-top: 15px;
+  line-height: 16px;
+}
+
+.title {
+  font-weight: 700;
```

Comment:
instead of setting values like this, use `<h3>` or something to that effect so it is friendlier to the a11y softwares.

Reply:
I can replace with h3 but then I need other additional handling (for margin)